MAINTAINERS

```diff
@@ -600,6 +616,11 @@ made through a pull request.
 	Email = "SvenDowideit@home.org.au"
 	GitHub = "SvenDowideit"
 
+	[people.thajeztah]
+	Name = "Sebastiaan van Stijn"
+	Email = "sebastiaan@gone.nl"
```

Comment:
Should this match the email I'm using for signing-off? In that case `github@gone.nl` :)